### PR TITLE
Change /crowd to /status

### PR DESCRIPTION
**Pull request by dns2utf8**
I'd change the /crowd command to /status.

That object would return all variable data from the sensors. We could also call it /sensors, but /status probably makes more sense.
- Open yes/no
- People present
- Temperature

Once we add more information, the status becomes more interesting.

After all, it doesn't take longer (maybe a few ms), doesn't need an additional request and improves the usefulness of the command.

What do you think, @dns2utf8?

### CODE REVIEW

**Comment by dns2utf8** [comment]
Ready for another review.

File: src/spaceapi_client.rs [hunk 1]
```diff
@@ -128,39 +128,87 @@ fn fetch_binary(url :&String) -> Result<Vec<u8>,io::Error> {
...
+  
+  if let Value(sensors) = status.sensors {
+    let pnp = match extract_sensors(sensors.people_now_present, "people_now_present") {
+      Ok(o) => {
+        o.into_iter().map(|e| {
+          /*let value_s = match e.value {
+            0 => format!("Coredump is closed\nNobody here right now."),
+            1 => format!("Coredump is open\nOne person is present!"),
+            people_now_present =>  format!("Coredump is open\n{} people are present!", people_now_present),
+          };*/
```
Comment (dbrgn) [inline src/spaceapi_client.rs]: Dead code!

File: src/spaceapi_client.rs [hunk 2]
```diff
@@ -128,39 +128,87 @@ fn fetch_binary(url :&String) -> Result<Vec<u8>,io::Error> {
...
+
+
+
+
+
+
+
+
+
+
```
Comment (dbrgn) [inline src/spaceapi_client.rs:201]: Lots of unnecessary whitespace.

File: src/spaceapi_client.rs [hunk 3]
```diff
@@ -184,24 +232,46 @@ mod test {
...
     
     assert_eq!( Ok("\nHackerspace: Coredump is open\n6 people are present!".into()), n );
   }
   #[test]
   fn extract_people_now_present_err() {
-    let e = extract_people_now_present( minimal_response().sensors.clone() );
+    let e = extract_people_now_present( minimal_response().sensors.unwrap() );
     
     assert_eq!( Err("response contains no sensors.people_now_present".into()), e );
+  }*/
```
Comment (dbrgn) [inline src/spaceapi_client.rs]: Dead code :)

File: src/user_input_compiler.rs [hunk 4]
```diff
@@ -442,8 +442,8 @@ mod test {
   }
   
   #[test]
-  fn crowd() {
-    assert_eq!(Status, Input::from( format!("/crowd") ) )
+  fn crowd() { // cancelled with 0.3.0
+    assert_eq!(InvalidSyntax("Invalid CommandWord".into()), Input::from( format!("/crowd") ) )
   }
```
Comment (dbrgn) [inline src/user_input_compiler.rs:447]: I think you could also remove the test.

Comment (dns2utf8) [inline src/user_input_compiler.rs:447]: I prefere it as a negative test, so it will not become implemented again by an accident.

File: src/spaceapi_client.rs (hunk 2 above)
Comment (dns2utf8) [inline src/spaceapi_client.rs:201]: This is separating the code from the tests.
